schedule/yast/rmt_server_install/rmt_server_install.yaml

```diff
@@ -0,0 +1,43 @@
+---
+name: rmt_server_install
+description: |
+  Perform an installation enabling some modules and selecting some
+  patterns.This test suite always registers to have access to some modules.
+vars:
+  YUI_REST_API: 1
+schedule:
+  - installation/bootloader_start
+  - installation/setup_libyui
+  - installation/product_selection/install_SLES
+  - installation/licensing/accept_license
+  - installation/registration/register_via_rmt
+  - installation/module_registration/register_nonconflicting_modules
+  - installation/add_on_product/skip_install_addons
+  - installation/system_role/accept_selected_role_SLES_with_GNOME
+  - installation/partitioning/select_guided_setup
+  - installation/partitioning/guided_setup/accept_default_part_scheme
+  - installation/partitioning/guided_setup/do_not_propose_separate_home
+  - installation/partitioning/accept_proposed_layout
+  - installation/clock_and_timezone/accept_timezone_configuration
+  - installation/authentication/use_same_password_for_root
+  - installation/authentication/default_user_simple_pwd
+  - installation/select_patterns
+  - installation/installation_settings/validate_default_target
+  - installation/bootloader_settings/disable_boot_menu_timeout
+  - installation/launch_installation
+  - installation/confirm_installation
+  - installation/performing_installation/perform_installation
+  - installation/logs_from_installation_system
+  - installation/performing_installation/confirm_reboot
+  - installation/handle_reboot
+  - installation/first_boot
+  - console/system_prepare
+  - console/consoletest_setup
+  - console/validate_repos
+test_data:
+  repos:
+    - uri: /repo/SUSE/Products/SLE-Module-Basesystem/15-SP4/x86_64/product/?credentials=Basesystem_Module_x86_64
```

Comment:
Put data contain "x86_64" keyword in here maybe not a good idea, how to handle it if we start load this yaml on other platform such as 390?
I suggest just using "zypper lr" and check output with key word such as SLE-Module-Basesystem etc in code directly.
But i have no idea is feasible for libyui case or not, this is new thing to me :)

Reply:
That's a valid point. I'll modify it so it gets the version and arch from the test vars.